docs/src/manual/complex.md

```diff
@@ -10,9 +10,7 @@ DocTestFilters = [r"≤|<=", r"≥|>=", r" == | = ", r" ∈ | in ", r"MathOptInt
 # Complex number support
 
 JuMP has a limited range of support for complex-valued variables and
-constraints. Since no current solvers natively support complex-valued variables
-and constraints, JuMP reformulates all complex expressions into their real and
-imaginary parts.
+constraints.
```

Comment:
If you remove the next sentence, it's not clear why you say "limited range"

Reply:
What else could be supported ?

Reply:
Suggestions then? Some more general introduction? I think it is limited because it's only Complex-in-EqualTo and HermitianPSD.

Reply:
Not sure what else we could support. You could say it's limited because the variables are actually real under the hood but at the JuMP level, you don't really see the difference. Maybe we can just remove the fact that it's limited. JuMP don't "reformulates all complex expressions into real and imaginary parts". Bridges only do that in case the solver does not support it which is in fact the best thing you could do. So it's actually the opposite, JuMP passes the complex expressions as is. Additionally, reformulation is done automatically for the solvers that need it

Reply:
I tried afresh: https://github.com/jump-dev/JuMP.jl/pull/3262